docs/permissions.md

```
@@ -1,73 +1,446 @@
+## 1. Concepts fondamentaux
+
+### 1.1 Modèle à deux niveaux
+
+Messages applique une autorisation à deux niveaux, indépendants mais composés :
 
-```python
-VIEWER = 1   # Read-only: view mailbox threads/messages
-EDITOR = 2   # Edit: create drafts, flag, delete, manage thread access
-SENDER = 3   # Send: EDITOR + can send messages
-ADMIN  = 4   # Admin: SENDER + manage mailbox accesses, labels, templates, import
 ```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)